packages/babel-plugin-class-features/test/fixtures/plugin-proposal-class-properties/loose-not-overwritten/options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": [
+    ["proposal-class-properties", { "loose": true }, "name 1"],
```

Comment:
what is supposed to happen here? I admit I don't quite get it

Reply:
When you specify the same plugin two times with different options, only the first set of options is considered because that instance of the plugin runs first.
This test ensures that this behvior is kept even with the merged plugin.

Reply:
oh, that's why those have ids (name 1/2) on them? otherwise the second one would just override first completely, right?

Reply:
IIRC Babel 7 throws when a plugin is duplicated in the same config file, unless you give them two different names.